File: Semestrul 4/MPP/Proiecte cursuri/ChatSockets/networking/UserDTO.cs
using System;

namespace chat.network.dto
{
	
	[Serializable]
	public class UserDTO
	{
		private string id;
		private string passwd;

		public UserDTO(string id) : this(id,"")
		{
		}

		public UserDTO(string id, string passwd)
		{
			this.id = id;
			this.passwd = passwd;
		}

		public virtual string Id
		{
			get
			{
				return id;
			}
			set
			{
				this.id = value;
			}
		}


		public virtual string Passwd
		{
			get
			{
				return passwd;
			}
		}
	}

}

File: Semestrul 4/MPP/Practice/HintGame/Networking/src/main/java/ro/mpp2024/networking/objectprotocol/dto/UserDTO.java
package ro.mpp2024.networking.objectprotocol.dto;

import ro.mpp2024.domain.User;

import java.io.Serializable;

public record UserDTO(int id, String alias) implements Serializable {
    @Override
    public String toString() {
        return "UserDTO{" +
                "id=" + id +
                ", alias='" + alias + '\'' +
                '}';
    }
}
